mlflow/tracking/client.py

```diff
@@ -734,7 +735,7 @@ def log_metric(
         value: float,
         timestamp: Optional[int] = None,
         step: Optional[int] = None,
-        synchronous: bool = True,
+        synchronous: Optional[bool] = None,
```

Comment:
I think the argument description should be updated.

Reply:
it should be fine, the original argument docstring doesn't contain the default value.

Reply:
actually some parts do need a change, let me fix